hw/ip/keymgr/data/keymgr.prj.hjson

```diff
@@ -11,8 +11,8 @@
       {
         version:            "0.1",
         life_stage:         "L1",
-        design_stage:       "D1",
-        verification_stage: "V1",
+        design_stage:       "D2",
```

Comment:
Please don't do two stage transitions in the same PR. The design one should be separate, as it needs a different review.

Reply:
Good point. Let me make another PR for that.